docs/bridge/light_client_prover.mdx

```diff
@@ -3,26 +3,37 @@ title: Prover Contract
 sidebar_position: 4
 ---
 
-What makes the Prover contract interesting is the ability to prove that something happened on a specific chain. The Prover takes as input the proof data that contain a merkle path to the block where the transaction/receipt originated and a merkle path to the transaction/receipt. Also, the height of the known block to the light client contract needs to be provided. This block needs to be ahead or on the block of the transaction that we are proving. With all of this the prover can calculate the expected block merkle root and compare it to the one stored in the light client block.
+The Prover contract proves that a transaction occurred on the opposite chain (for example on the Near side if this Prover contract is on the Calimero side). Prover contract asks light client contract for the Merkle tree root for the specific block height, and he uses this information for the verification of the proof.
```

Comment:
Can you please somehow include this text here as well:

`Proof for a certain receipt can be fetched by making an RPC call (EXPERIMENTAL_light_client_proof) to the Archival node. Together with the block height from the light client contract which needs to be higher than the block height of the block where the transaction/receipt happened, everything that needs to be provided to the Prover is ready.`

This is deleted in this PR, but I think that it has value